extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/AbstractBaseJob.java

```diff
@@ -8,41 +8,47 @@
  */
 package org.eclipse.smarthome.binding.astro.internal.job;
 
-import org.quartz.Job;
-import org.quartz.JobDataMap;
-import org.quartz.JobExecutionContext;
-import org.quartz.JobExecutionException;
+import java.text.SimpleDateFormat;
+import java.util.HashMap;
+import java.util.Map;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Baseclass for all jobs with common methods.
  *
  * @author Gerhard Riegler - Initial contribution
+ * @author Christoph Weitkamp - Removed Quartz dependency
  */
-public abstract class AbstractBaseJob implements Job {
+public abstract class AbstractBaseJob implements Runnable {
     private final Logger logger = LoggerFactory.getLogger(AbstractBaseJob.class);
+    public static final SimpleDateFormat ISO8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
```

Comment:
I maybe observed an issue with the `DateExpression` from ESH scheduler. Every schedule is executed more than once. I am going to file an issue for that if I can confirm my guess.

Currently I want to replace `DateExpression` completely with `CronExpression`.